Makefile

```diff
@@ -13,8 +13,6 @@ build:
 
 export TRANSIFEX_RESOURCE = paragon
 transifex_langs = "ar,ca,es_419,fr,he,id,ko_KR,pl,pt_BR,ru,th,uk,zh_CN"
-i18n = ./src/i18n
```

Comment:
[curious] Are we actually able to be remove `i18n` given it's referred in `detect_changed_source_translations` below?

```shell
detect_changed_source_translations:
	# Checking for changed translations...
	git diff --exit-code $(i18n)
```

That said, I'm not sure if `detect_changed_source_translations` is actually used either.

Reply:
Good note @adamstankiewicz. I've removed the step since it'll be removed when the OEP-58 is fully implemented.

Reply:
@OmarIthawi are you certain that is an appropriate solution here? Can you confirm this won't break the current translation flow?

Reply:
Good note. It won't break this repository, but I'm not so sure about the other workflows: https://github.com/search?q=org%3Aopenedx%20detect_changed_source_translations&type=code

It's safer to close this PR until we have the new OEP-58 workflow set up.